polaris-tokens/src/token-groups/shape.ts

```diff
@@ -74,6 +77,7 @@ export const shapeBorderRadiusScale = [
   '4',
   '5',
   '6',
+  'full',
```

Comment:
Adding this value here. I see it as a part of the scale and not an alias. Let me know if there are any problems.